api/zeebe/zeebe-openapi.yaml

```diff
@@ -0,0 +1,386 @@
+openapi: "3.1.0"
+info:
+  title: Zeebe REST API
+  version: "0.1"
+  description: API for communicating with the Zeebe cluster.
+  license:
+    name: Zeebe Community License Version 1.1
+    url: https://github.com/camunda/zeebe/blob/main/licenses/ZEEBE-COMMUNITY-LICENSE-1.1.txt
+externalDocs:
+  description: Find out more
+  url: https://docs.camunda.io/docs/apis-tools/zeebe-api-rest/overview/
+
+servers:
+  - url: "{schema}://{host}:{port}/v1"
+    variables:
+      host:
+        default: localhost
+        description: The hostname of a Zeebe Gateway.
+      port:
+        default: "8080"
+        description: The port of the Zeebe REST API server.
+      schema:
+        default: http
+        description: The schema of the Zeebe REST API server.
```

Comment:
When this is released, will it be updated to match the other Explorers with `SERVER-URL`?

Reply:
Actually I think the opposite -- I think I'm going to change all the other ones to use something like this, because I really like the way it plays out in the explorer.

From https://loquacious-pika-a05ac2.netlify.app/api/zeebe/docs/complete-a-user-task/, on the right side you have a "Base URL". If you hover over it, you get an "Edit" button. Clicking it expands the individual components of the URL, and you can then each component and click "Hide" to save the changes. Once you've done that, all examples with the explorer now use your new base URL.

It's not totally intuitive, but I think it's much better than the current experience with `SERVER-URL`, because you can't change that at all in the other explorers. (You can _try_, by clicking the "Edit" URL, but you can't actually change anything.)

If there's a concern about using `localhost` as the default, maybe there's some intermediate option to use instead, but I like the customizability this brings.

Reply:
Ok great, I'm just looking for consistency either way at this point.

SERVER-URL isn't my favorite, but it was difficult to communicate what the SaaS and SM URLs could look like. If we can do that (or make the fields editable) and make this explorer actually send requests, that would be very, very cool. 

Sounds like you have this handled, though; carry on!

Reply:
Yeah that SERVER-URL nonsense was me wanting to scrub the generated URL from when I downloaded the spec, but not really grasping how good the experience could be with proper spec definition.